lib/util/webhook.js

```diff
@@ -8,15 +8,35 @@ const parseString = require("xml2js").parseString;
 const WEBHOOK_URL =
   "https://getin-server.herokuapp.com/api/v1/mapping_encounter_webhook";
 
+const BACKEND_URL = "http://localhost:8383";
+
 // Recieves and Instance ID, a form ID and a project ID then retrieves the xml attached, converts it to a json object and pushes it to our backend service
 const pushToWebhook = (instanceId, formId, projectId) => {
-  const submisionURL = `https://localhost:8989/v1/projects/${projectId}/forms/${formId}/submissions/${instanceId}.xml`;
+  const submisionURL = `${BACKEND_URL}/v1/projects/${projectId}/forms/${formId}/submissions/${instanceId}.xml`;
+
+  // Get access token to use
+  const getAccessToken = request(
+    {
+      method: "POST",
+      url: `${BACKEND_URL}/v1/sessions`,
+      headers: {
+        "Content-Type": "application/json"
+      },
+      // Find a better way of parsing these credentials.
+      body: '{  "email": "pkigenyi@outbox.co.ug",  "password": "Doppler25"}'
```

Comment:
I think we could put these in environment variable

Reply:
I agree. Not sure how to do that yet but wanted to test these the way they are to be sure nothing is broken for now